### DISCUSSION TITLE
Richtext box related query

### DISCUSSION THREAD

**Discussion by krishbbc** [GrapesJS/grapesjs#6350] [General]
In that reachtext box we have to add each controls by writing a code or one by one ,
so is there any plugin for richtextbox which have some default controls in grapesjs.

**Comment by danstarns** [comment]
> we have to add each controls by writing a code or one by one

This is the way GrapesJS is designed, we give you building blocks to expand out into your application's needs. 

> is there any plugin for richtextbox which have some default controls in grapesjs.

Checkout: 

* https://github.com/GrapesJS/ckeditor
* https://github.com/Ju99ernaut/grapesjs-rte-extensions
* https://grapesjs.com/docs/guides/Replace-Rich-Text-Editor.html

Please give your specific setup and we can work on getting the controls working for you from there.
